cannot deploy a single template to org-formation controlled accounts

i’m testing my new template and I would like to use orgf to deploy it to my account however there doesn't seem to be a way to deploy just one template to an account.  i'm using orgf ver 0.9.14 and node ver 15.5.1


This didn’t seem to work for me..

```
npx org-formation update-stacks 900-testing/orgf-noop.yaml --profile testitorgmgnt-CiServiceRole --stack-name orgf-noop
ERROR: unable to load file 900-testing/orgf-noop.yaml.
reason: Top level Organization attribute is missing.
npm ERR! code 1
npm ERR! path /Users/zaro0508/work-sage/testitorgmgnt-infra/org-formation
npm ERR! command failed
npm ERR! command sh -c org-formation update-stacks 900-testing/orgf-noop.yaml --profile testitorgmgnt-CiServiceRole --stack-name orgf-noop

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/zaro0508/.npm/_logs/2021-02-08T23_22_01_049Z-debug.log
```

The error seems to indicate that update-stacks needs to know about the accounts to deploy to, which is defined in a separate organization.yaml.  However there's no way to tell update-stacks about the accounts because the update-stacks command does not have an  `--organization-file` flag.

--- a/docs/cli-reference.md
+++ b/docs/cli-reference.md
@@ -105,6 +105,7 @@ Will deploy CloudFormation resources specified in *templateFile*.
 |<nobr>--parameters</nobr> | none | parameters that need to be passed to the CloudFormation template.|
 |<nobr>--termination-protection</nobr> | false | If specified the stack will be created with termination protection.|
 |<nobr>--update-protection</nobr> | false | When set to `true` will create a StackPolicy for the stacks that prevents any resource from being modified through CloudFormation. |
+|<nobr>--organization-file</nobr>| undefined | path to the organization file that will be used to evaluate organizational bindings |
 |<nobr>--max-concurrent-stacks</nobr> | 1 | Maximum number of stacks to be updated concurrently |
 |<nobr>--failed-stacks-tolerance</nobr> | 0 | The number of failed stacks after which execution stops|
 
@@ -125,6 +126,7 @@ validates the CloudFormation templates that will be generated based on *template
 |option|default|description|
 |---|---|---|
 |<nobr>--stack-name</nobr> | 'validation' | The stack name used to deploy CloudFormation resources (used in e.g. generated names for output)|
+|<nobr>--organization-file</nobr>| undefined | path to the organization file that will be used to evaluate organizational bindings |
 |<nobr>--parameters</nobr> | none | parameters that need to be passed to the CloudFormation template.|
 
 

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "aws-organization-formation",
-  "version": "0.9.15-beta.7",
+  "version": "0.9.15-beta.8",
   "description": "Infrastructure as code solution for AWS Organizations",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",

--- a/src/commands/update-stacks.ts
+++ b/src/commands/update-stacks.ts
@@ -58,6 +58,7 @@ export class UpdateStacksCommand extends BaseCliCommand<IUpdateStacksCommandArgs
         command.option('--stack-description [description]', 'description of the stack that will be displayed CloudFormation');
         command.option('--parameters [parameters]', 'parameter values passed to CloudFormation when executing stacks');
         command.option('--termination-protection', 'value that indicates whether stack must have deletion protection');
+        command.option('--organization-file [organization-file]', 'organization file used for organization bindings');
         command.option('--update-protection', 'value that indicates whether stack must have a stack policy that prevents updates');
         command.option('--max-concurrent-stacks <max-concurrent-stacks>', 'maximum number of stacks to be executed concurrently', 1);
         command.option('--failed-stacks-tolerance <failed-stacks-tolerance>', 'the number of failed stacks after which execution stops', 0);

--- a/src/commands/validate-stacks.ts
+++ b/src/commands/validate-stacks.ts
@@ -24,6 +24,7 @@ export class ValidateStacksCommand extends BaseCliCommand<IUpdateStacksCommandAr
 
     public addOptions(command: Command): void {
         command.option('--parameters [parameters]', 'parameter values passed to CloudFormation when executing stacks');
+        command.option('--organization-file [organization-file]', 'organization file used for organization bindings');
         command.option('--stack-name <stack-name>', 'name of the stack that will be used in CloudFormation', 'validation');
         super.addOptions(command);
     }

